fastly/fastly_test_utils.go

```diff
@@ -55,13 +61,21 @@ func computeServiceIDForTest() string {
 	return defaultComputeTestServiceID
 }
 
+func ngwafWorkspaceIDForTest() string {
+	if tsid := os.Getenv("FASTLY_TEST_NGWAF_WORKSPACE_ID"); tsid != "" {
+		return tsid
+	}
+
```

Comment:
lol, this is such a nit I'm almost embarrassed to comment 😅 but could we lose the line break between the the if statement and the return as we don't need the breathing space (IMHO).
```suggestion
```